package.json

```diff
@@ -4,18 +4,14 @@
   "description": "Open Data Portal - French customization",
   "author": "Etalab",
   "scripts": {
-    "dev": "npm run clean && npm run watch",
-    "build": "npm run clean && npm run build:stylemark && npm run build:app",
-    "build:app": "parcel build theme/less/index.less theme/js/index.js --dist-dir udata_front/theme/gouvfr/static/",
-    "build:stylemark": "npm run build:app -- --no-optimize",
+    "dev": "vite build --watch",
```

Comment:
I think this is due to how the package is loaded.

 Its `package.json` references the file `dist/index.js`  that doesn't exist when you switch to the branch. The package `udata-front/udata-front` must be built before the `npm install` of `udata-front`.

This should be resolved when we move to a package hosted on NPM because we'll publish the `dist` directory.

Reply:
it's fixed, CI is able to install the deps without error